app/src/androidTest/java/org/mozilla/tv/firefox/ui/robots/NavigationOverlayRobot.kt

```diff
@@ -75,12 +75,25 @@ class NavigationOverlayRobot {
             return BrowserRobot.Transition()
         }
 
-        fun openTileToBrowser(index: Int) {
+        fun openPocketMegatile(interact: PocketScreenRobot.() -> Unit): PocketScreenRobot.Transition {
+            pocketMegaTile().click()
+
+            PocketScreenRobot().interact()
+            return PocketScreenRobot.Transition()
+        }
+
+        fun openTileToBrowser(index: Int, interact: BrowserRobot.() -> Unit): BrowserRobot.Transition {
             homeTiles().perform(RecyclerViewActions.actionOnItemAtPosition<TileViewHolder>(index, click()))
+
+            BrowserRobot().interact()
+            return BrowserRobot.Transition()
         }
 
-        fun openTileToBrowser(title: String) {
+        fun openTileToBrowser(title: String, interact: BrowserRobot.() -> Unit): BrowserRobot.Transition {
             homeTiles().perform(RecyclerViewActions.actionOnItem<TileViewHolder>(hasDescendant(withText(title)), click()))
+
+            BrowserRobot().interact()
```

Comment:
Seems like it could potentially be a good idea.  Then again, the total duplication is relatively small, and it's not the kind of duplication I normally worry about (that is, these aren't implementations that are going to fall out of sync with one another and cause subtle bugs).  I'm also concerned about putting more abstraction than necessary into our tests.

Overall, I'm ambivalent on this one.